src/test/runTest.ts

```diff
@@ -17,14 +27,32 @@ async function main() {
         const extensionTestsPath = path.resolve(__dirname, './suite/index');
         const vscodeExecutablePath = await downloadAndUnzipVSCode('insiders');
         const cliPath = resolveCliPathFromVSCodeExecutablePath(vscodeExecutablePath);
+        const extensionsDir = await getExtensionsDir();
+
         console.info(`Installing Jupyter Extension`);
-        spawnSync(cliPath, ['--install-extension', 'ms-toolsai.jupyter', '--disable-telemetry'], {
-            encoding: 'utf-8',
-            stdio: 'inherit'
-        });
+        spawnSync(
+            cliPath,
+            [
+                '--install-extension',
+                'ms-toolsai.jupyter',
+                '--pre-release',
+                '--extensions-dir',
+                extensionsDir,
```

Comment:
This is what was required to get the tests working when using `extensionDependencies`